### PR TITLE
request invite code page. removing flex 1 from fullwidth buttons

### PR DESCRIPTION
**Pull request by chrisnojima**
@keybase/react-hackers

### CODE REVIEW

File: shared/common-adapters/button.native.js [hunk 1]
```diff
@@ -77,9 +77,7 @@ const commonLabel = {
   textAlign: 'center'
 }
 const fullWidth = {
-  flex: 1,
```
Comment (chrisnojima) [inline shared/common-adapters/button.native.js]: this just depends too much on the parent context so i'm removing it

File: shared/login/signup/request-invite.js [hunk 2]
```diff
@@ -6,17 +6,34 @@ import {bindActionCreators} from 'redux'
 
 import Render from './request-invite.render'
 import * as signupActions from '../../actions/signup'
+import type {Props} from './request-invite.render'
 
 class RequestInvite extends Component {
+  state: {
```
Comment (chrisnojima) [inline shared/login/signup/request-invite.js]: hoisting this up to the smart component

File: shared/login/signup/request-invite.render.native.js [hunk 3]
```diff
@@ -1,16 +1,62 @@
...
+          disabled={!this.props.email} />
+        <Box style={{flex: 1}} />
+      </Container>
+    )
   }
 }
 
+const stylesButton = {
+  marginTop: 50
+}
```
Comment (cjb) [inline shared/login/signup/request-invite.render.native.js]: Totally trivial, but shall we decide on whether there's an empty line between each styles object? I vote no. :)

File: shared/login/signup/request-invite.render.native.js [hunk 4]
```diff
@@ -1,16 +1,62 @@
...
+const stylesHeader = {
+  marginTop: 30,
+  alignSelf: 'center'
+}
+const stylesIcon = {
+  marginTop: 40,
+  alignSelf: 'center'
+}
+const stylesInput = {
```
Comment (cjb) [inline shared/login/signup/request-invite.render.native.js]: Any reason to keep this?

**Comment by cjb** [comment]
:thumbsup:

File: shared/login/signup/request-invite.render.native.js (hunk 3 above)
Comment (chrisnojima) [inline shared/login/signup/request-invite.render.native.js]: yeah we should discuss. i wish we could force eslint.. maybe we can. also we might have to use StyleSheet.create() on mobile, unclear if we're paying a serious penalty or not , not using it

**Pull request merged**